src/CMakeLists.txt

```diff
@@ -99,6 +99,13 @@ if(CELERITAS_USE_Geant4)
   add_subdirectory(accel)
 endif()
 
+if(CELERITAS_USE_VecGeom AND CELERITAS_USE_Geant4)
+  celeritas_target_include_directories(celeritas
+    PUBLIC
+    $<BUILD_INTERFACE:${Geant4_INCLUDE_DIRS}>
+  )
+endif()
+
```

Comment:
This needs to be reverted. The fix for building was inside `src/celeritas`.

Reply:
Reverted.